.github/workflows/pr-test.yml

```
@@ -0,0 +1,44 @@
+        run: ./gradlew clean test
+
+      - name: if fail
+        if: failure()
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const pull_number = context.payload.pull_request.number;
+            const title = context.payload.pull_request.title;
+            const updated_title = `❌[BUILD FAIL] ${title}`;
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)